core/io/image.cpp

```diff
@@ -3139,6 +3164,10 @@ void Image::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("save_png", "path"), &Image::save_png);
 	ClassDB::bind_method(D_METHOD("save_png_to_buffer"), &Image::save_png_to_buffer);
 	ClassDB::bind_method(D_METHOD("save_exr", "path", "grayscale"), &Image::save_exr, DEFVAL(false));
+#ifdef MODULE_WEBP_ENABLED
+	ClassDB::bind_method(D_METHOD("save_webp", "path", "quality"), &Image::save_webp, DEFVAL(-1.0f));
+	ClassDB::bind_method(D_METHOD("save_webp_to_buffer", "quality"), &Image::save_webp_to_buffer, DEFVAL(-1.0f));
+#endif
```

Comment:
These should always be defined, the API shouldn't depend on module config for some as core as Image. We already do the same for PNG and EXR, it should just throw an error if `save_webp_func` is not enabled (that's why it's a function pointer, and not a direct reference to `ImageSaverWebP` or similar).